code/datums/wounds/bones.dm

```diff
@@ -0,0 +1,282 @@
+
+/*
+	Bones
+*/
+// TODO: well, a lot really, but i'd kill to get overlays and a bonebreaking effect like Blitz: The League, similar to electric shock skeletons
+
+/*
+	Base definition
+*/
+/datum/wound/brute/bone
+	sound_effect = 'sound/effects/crack1.ogg'
+	wound_type = WOUND_TYPE_BONE
+
+	/// The item we're currently splinted with, if there is one
+	var/obj/item/stack/splinted
+
+/*
+	Overwriting of base procs
+*/
+/datum/wound/brute/bone/apply_wound(obj/item/bodypart/L, silent=FALSE, datum/wound/old_wound = null)
+	. = ..()
+	if(QDELETED(src) || !limb)
+		return
+	RegisterSignal(victim, COMSIG_HUMAN_EARLY_UNARMED_ATTACK, .proc/attack_with_hurt_hand)
+	if(L.held_index && victim.get_item_for_held_index(L.held_index) && (disabling || prob(30 * severity)))
+		var/obj/item/I = victim.get_item_for_held_index(L.held_index)
+		if(istype(I, /obj/item/offhand))
+			I = victim.get_inactive_held_item()
+
+		if(I && victim.dropItemToGround(I))
+			victim.visible_message("<span class='danger'>[victim] drops [I] in shock!</span>", "<span class='warning'><b>The force on your [L.name] causes you to drop [I]!</b></span>", vision_distance=COMBAT_MESSAGE_RANGE)
+
+	update_inefficiencies()
+
+/datum/wound/brute/bone/remove_wound(ignore_limb, replaced)
+	limp_slowdown = 0
+	if(limb)
+		REMOVE_TRAIT(limb, TRAIT_LIMB_DISABLED_WOUND, src)
+	if(victim)
+		UnregisterSignal(victim, COMSIG_HUMAN_EARLY_UNARMED_ATTACK)
+	return ..()
+
+// note this is only for humans since they alone have COMSIG_HUMAN_EARLY_UNARMED_ATTACK (obviously)
+/datum/wound/brute/bone/proc/attack_with_hurt_hand(mob/M, atom/target, proximity)
```

Comment:
why not make this apply when you're attacking with an item that's held in that hand as well?